src/main/java/net/neoforged/neoforge/registries/callback/BakeCallback.java

```diff
@@ -0,0 +1,16 @@
+package net.neoforged.neoforge.registries.callback;
+
+import net.minecraft.core.Registry;
+
+/**
+ * Fired when the registry is finished with all registration.
```

Comment:
Does this really only fire when registration is done or does it also fire on snapshot injection (which it did for `ForgeRegistry`s if I recall correctly)?